src/components/error-dialog/index.tsx

```diff
@@ -0,0 +1,35 @@
+import * as React from 'react';
+
+import { IconButton, Button } from '@zero-tech/zui/components';
+import { IconXClose } from '@zero-tech/zui/icons';
+
+import { bemClassName } from '../../lib/bem';
+
+import './styles.scss';
+
+const cn = bemClassName('error-dialog');
+
+export interface Properties {
+  onClose?: () => void;
+}
+
+export class ErrorDialog extends React.Component<Properties> {
+  render() {
+    return (
+      <div {...cn('')}>
+        <IconButton {...cn('close')} size={40} Icon={IconXClose} onClick={this.props.onClose} />
+
+        <div {...cn('heading-container')}>
+          <div {...cn('header')}>Access Denied</div>
+          <div {...cn('divider')} />
```

Comment:
Do we need a "divider"? Why can't the `header` or the `heading-container` just have the border?

Reply:
Yeah I can update one of the header borders instead - admittedly this was just a copy and paste from one of the other modals.

Reply:
We definitely need to generalize this modal a bit. We shouldn't need to rebuild this whole thing every time. lol